src/components/Tabs/Tabs.tsx

```diff
@@ -159,16 +161,20 @@ export const Tabs = ({
   }, [prevActiveIndex, activeIndex, tabRefs]);
 
   /**
-   * Autogenerate ids on tabs if not defined
+   * Autogenerate ids on tabs if not defined.
    */
   useEffect(() => {
-    setId(tabs().map((tab) => (tab.props.id ? tab.props.id : nanoid())));
+    setId(
+      tabs().map((tab) => (tab.props.id ? tab.props.id : uidSeed(`${tab}-id`))),
+    );
     setAriaLabelledBy(
       tabs().map((tab) =>
-        tab.props.ariaLabelledBy ? tab.props.ariaLabelledBy : nanoid(),
+        tab.props.ariaLabelledBy
+          ? tab.props.ariaLabelledBy
+          : uidSeed(`${tab}-aria-labelledby`),
       ),
     );
-  }, [tabs]);
+  }, [tabs, uidSeed]);
```

Comment:
documentation for `useUIDSeed`: https://github.com/thearnica/react-uid#hooks-168

this is 1 case where `nanoid` feels a little cleaner, but logic this complicated should be rare